content/md/franklin-insights/franklin-insights-getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+id: franklin-insights-getting-started
+themes: franklin-insights
+title:  Franklin Insights **getting started**
+ee-only: true
+related: franklin-insights-mappings, franklin-insights-subscriptions
+---
+
+# Overview
+
+Welcome to Franklin’s world! Franklin is our new **product data intelligence assistant**, at your service :wink:
+
+Franklin helps Julia on her catalog enrichment mission and guides her towards creating a **more compelling product experience, faster than ever before**.
```

Comment:
We prefer to directly say "you" instead of talking about Julias. Our readers are Julias. ;)